examples/step-49/step-49.ipynb

```diff
@@ -0,0 +1,1128 @@
whAPTs2TPeMU6YMIGcnJx45YiIXr9+zRWDF4rdu3dXKGZuDF988QXNnTtXcNvvCjVr1uTN7Th69CgxDEPJycmC2+3evTvJZDISiUTk6ekpKHu3uLiY5HI59evXT3A/ZcfYvHlzrrA3APL19X3jrOb169cTy7J06dIlKi4upqpVq5KVlRU9fvzYYN8sy9LWrVtp2LBhFBISQvb29lxuhq4AeatWrWjevHl06dKlCuP68f/Y++6wKM6u/Xu2sMsCS5PekaaAgg1FbGBsGKwY0bz2xBIN9liiRqPJq2hUjBVr1BiiJpIYMSqKLUqUWEBFEQRFAem9bDm/P8jOD4TdGZP4vvm+L/d1zcUyc+Z5ZnZnzjxznvvcZ+9ezryYU6dO8eLFMwxD9+/f12kTFBREffr0abb+zp07tHr1ahowYAA5Ojqy36VMJiNPT08aPXo07d69u8n99l/l4RNRX23bGIbJZxjGhhpG9zYAWiS1MgwjBnAcwGEi+o6rz/8kuEa9fOl+r0O35BNaUSgUkMlkaNOmjU679PR0FBcXc8aw58yZg9jYWHz//fdIT09HRkYGsrOzWR53SkoKysvL2diphYUF9PX1YW5uDgcHB3h5eaFDhw4ICgpCu3btIBAIUFxczJlBqYGmmMnrxPz79euH+vp6VFRUcOYF9OnTh800/W8hPz8fOTk58Pf312k3efJkBAcH66wn2xgJCQk4ceIEfv75Z/j7+yMkJATu7u5YuXKlzvMdO3YsBAKB1iIpuhASEoKQkBDcv38fvr6+UKvVePz4MWQyGTp37oyZM2ciPDyc10j/xYsXmD59OubPn48OHToAaAjB+Pr6suGhGzduID4+Hjdv3kR2djbUajUmTpwIW1tbeHp64r333kO/fv3QvXt39g13wYIFWuvNOjo6co7ebW1tecXnhUIhCgoKdN6LUqkUT548wcyZM3Hjxg1kZGSwekImJiZwdXVF//79ERMTg7t37+osLflGoe1JwGcBEAVg0e+fFwFY14INA+ArAJtet/3/xAg/NTVVp83169dJKBRytrVy5Upq1aoVp114eDj5+Phw2vHR8CAi6t27Ny+7SZMmUevWrTntMjIyiGEYevLkCR04cIBmzpxJwcHB5OrqSkZGRmy2rFQqJT09PZJKpbRu3TrKz8/X2e6vv/7K603pVejp6dHx48c57X788UcSiURUWFj42n38Vfj4449JT0+Pli1bpvVN5ujRoyQQCFoc2bYEhUJBcrmchgwZ0mT9unXrSCgUUtu2bSk3N7fZftevXyeGYeinn356/RP5HY8ePSKpVEqBgYEkFovp22+/pZ07d1KnTp1IJBKRSCSiTp060c6dO3W+ubm7u5OHh0eL5+bk5EQASCKRkJubG4WHh9PevXvJzMyMli9frrVNuVxO0dHRWrdnZGRwZsgXFBTwyqKXSCR09OhRImrICL5y5QqtWLGCBgwYQM7OziSVStmRu52dHfXt25eWLVtGv/76a7M3Dz4+502O8P+swzdHAzsn/fe/Zr+vtwVw6vfPQb9/GXcB3P59GcSn/f+Ew+d6Vbty5Qovh79s2TKysLDgtBs2bBi1b9+e0y4wMJBCQkI47bp27UpvvfUWp114eDh5e3tz2iUmJpJIJNJpk52dTQcPHiRLS0vS19dnxaAMDAyoY8eOtGjRombf68mTJ3m9Pr8KOzs7mjVrlk6bO3fukEgkIjs7O5ozZ85r9/FXoKamhhwdHWn48OGkr69Penp69O6771JxcXETO2traxo6dCjvdkeMGEFGRkZUV1fXbFt2dja1bt2aRCIRbdu2jV2vUqnI0tKyxRADX2RmZpK+vj516dKFVCoVubm50ejRo5v08c0331CPHj1IIpGQQCAgX19f2rBhA9XU1LB28+fPJ7FY3OJDiYioW7du1L1792br3dzcaMKECVqPz87OjmbPnq11u0KhIABNjuVVqFQqAtBEDqLx/pcvX6ZPP/2UhEIhtWrVigwNDdmwqLm5OXXs2JEmTZpE+/fvp6FDh3JKUBC1LEXyKv62Dv9NL/8Jh9+S8l5jXL58mZfD56tuOXjwYOJzXp06daKBAwdy2nXo0IHCwsI47QYOHEidO3fmtDt06BDp6+tz2hE1KFpqnEBRURFFR0dT3759qVWrVgSA9PT0yNPTk9577z1asGABL62hV9GzZ88WHYIGWVlZJJVKqU+fPpSXl0fm5uaUkZHx2v38WURFRbG/g0qlog0bNpClpSUJBALq3bs33b9/n2JiYkgoFFJJSQmvNhMSEohhGDp16pROu0WLFpFAIKCAgAAqKyuj2bNnk0QioYqKij90LllZWSSTyahDhw7sCHXChAnk5OSkdZ+ffvqJ+vXrRzKZjBiGIXd3d5o6dSoBoN27d2vdz8jIiNatW9dsfUBAgM6BjK+vLw0fPlznefAZ0AGg9evX0wcffEA9evQgBwcHdsQuFArJzMyMGIYhb29v2rJli9Z5l4kTJ5Kbm5vOvogaHH5SUpJOmzfp8P/JtOWIQfKNzf/V/HqlUsmrzGBdXZ1OrXENKisrecXb8/LyeJc3LCkpYemBZmZmmDVrFs6ePYuCggLU1dXhq6++go+PD+Lj47F+/XpUV1cjIiKCU+e+MTp27IjMzMwWtxUXF6N9+/Zwd3fHuXPnYGVlhcjISCxZsoR3+38FiouLsXbtWqxduxZAwzU1d+5c5OfnIy4uDrm5ufD29sb777+P/v3788pxUCqVGD58OEJDQzFw4ECdtp9//jnu3r2Lp0+fwsLCAps3b0Z0dDSnLEVLyMnJgbe3N9zc3HDjxg32mo6IiEBOTo7W/QYNGoSff/4ZVVVVuHTpEtq2bYtdu3YBAEsvvXTpUpN9CgsLUVFR0WLhE2tra506N5aWlpxF7kUiEZ48eYKUlBTs3LkT06ZNQ+/eveHq6gq5XM7eh4sXL0ZcXByABgrr7t27kZubC6VSydYlHj58OGbOnKl13kUqlfKa72MYhpc0xJ
```

Comment:
This line is not necessary

Reply:
You mean you don't want the rendered images to be present in the notebook?

Reply:
Hmm I don't know why all my comments are on the line 337, looks like a bug in github. Several times you do something like:
```python
triangulation = Triangulation('2D')
triangulation = create_triangulation()
```
The first line is useless.